pkg/user/RoleGroupService.go

```diff
@@ -85,19 +85,26 @@ func (impl RoleGroupServiceImpl) CreateRoleGroup(request *bean.RoleGroup) (*bean
 		//loading policy for safety
 		casbin2.LoadPolicy()
 		//create new user in our db on d basis of info got from google api or hex. assign a basic role
+
 		model := &repository2.RoleGroup{
 			Name:        request.Name,
 			Description: request.Description,
 		}
 		rgName := strings.ToLower(request.Name)
 		object := "group:" + strings.ReplaceAll(rgName, " ", "_")
+
+		_, err := impl.roleGroupRepository.GetRoleGroupByCasbinName(object)
```

Comment:
are we only using this for checking if entry exists or not? if yes then we should use exists clause since this is confusing. Also, add comments